lib/ansible/modules/storage/netapp/sf_facts.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/python
+
+# (c) 2017, NetApp, Inc
+#
+# This file is part of Ansible
```

Comment:
Can you please replace line 4 to 19 to match a single liner GPL license (which is new in Ansible Module development)? 


```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```